B827EBFFFE1BFBFADBFAD7.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE1BFBFADBFAD7",
```

Comment:
This EUI is too long to be valid.

Reply:
i updated the name and the gateway EUI. thank you